addons/gsri_opex/functions/fremm/fn_fremmAddBoatBays.sqf

```diff
@@ -34,7 +34,7 @@ private _actionsList = [];
 {
 	private _name = getText (configFile >> "CfgVehicles" >> _x >> "displayName");
 	_actionsList pushBack ([format["action_%1",_x],_name,"",{_this spawn GSRI_fnc_bayReplace},{true},{},[_x]] call ace_interact_menu_fnc_createAction);
-} forEach ["B_Boat_Transport_01_F", "C_Boat_Transport_02_F", "B_Boat_Armed_01_minigun_F", "B_SDV_01_F"];
+} forEach ["B_Boat_Transport_01_F", "C_Boat_Transport_02_F", "B_Boat_Armed_01_minigun_F", "B_SDV_01_F", "I_C_Boat_Transport_02_F"];
```

Comment:
Could we move that class list to config ?
I'm considering it for helicopters also.

Reply:
I'd suggest opening an issue for this topic : this have to be reworked, in order to avoid duplicated entries. For example in this case, `C_Boat_Transport_02_F` and `I_C_Boat_Transport_02_F` will result in having to entries named `RHIB` on the bay's command, the only difference being the texture. I would like to have only one entry by boat type, and subchoices for available texture sources.

For choppers, having the same texture system would be nice, but we have a more important concern : we have to deal with helicopters on skate instead of wheels, that have to be spawned on mobile landing platforms to be towed out of the hangar.

Consequently, I'd say two issues for those two features, going for project 2020-06

Reply:
Opened #41 #42